NewRelic/Config.php

```diff
@@ -27,7 +27,7 @@ class Config
     private $customParameters;
     private $deploymentNames;
 
-    public function __construct(string $name, string $apiKey, string $licenseKey = null, bool $xmit = false, array $deploymentNames = [])
+    public function __construct(?string $name, string $apiKey = null, string $licenseKey = null, bool $xmit = false, array $deploymentNames = [])
     {
         $this->name = !empty($name) ? $name : \ini_get('newrelic.appname') ?: '';
         $this->apiKey = $apiKey;
```

Comment:
You need to update the return value of `getApiKey`

Reply:
good catch. Fixed